src/flask_container_scaffold/util.py

```diff
@@ -45,3 +50,30 @@ def _parse_cfg(config):
             settings_dict[key] = config[section][key]
         config_dict.update({section: settings_dict})
     return config_dict
+
+
+def parse_input(logger, obj):
+    """
+    Parses incoming request, returns a serializable object to return
+    to the client in all cases. When there is a failure, the
+    object contains error information.
+    :param Logger logger: Instantiated logger object
+    :param BaseModel obj: An object based ona a pydantic BaseModel to
+                          attempt to parse.
+    :returns: Instantiated object of type obj on success, or QueryResult
```

Comment:
question:  Is the reference to QueryResult accidental here? I don't see any QueryResult in the code.

Reply:
Yes, accidentally left that, I extracted this variant from your downstream project, since it was the most complete one we had, and missed this reference.